lib/krane/template_sets.rb

```diff
@@ -37,23 +37,30 @@ def ejson_secrets_file
         end
       end
 
-      def validate
+      def validate(render_erb: true)
         errors = []
+        supported_extensions = if render_erb
+          TemplateSets::VALID_TEMPLATES
+        else
+          TemplateSets::VALID_TEMPLATES.select { |extension| !extension.include?('erb') }
```

Comment:
Maybe we can use this instead?

```suggestion
          TemplateSets::VALID_TEMPLATES.reject { |extension| extension.include?('erb') }
```